packages/manager/apps/order-tracking/index.js

```diff
@@ -0,0 +1,41 @@
+import 'script-loader!jquery'; // eslint-disable-line
+
+import angular from 'angular';
+import ovhManagerCore from '@ovh-ux/manager-core';
+import ssoAuth from '@ovh-ux/ng-ovh-sso-auth';
```

Comment:
Is this dependency really needed?

Reply:
yes in order to be redirected to the login screen if you're not logged when starting the app